api/src/domain/license-status/searchLicenseStatusFactory.test.ts

```diff
@@ -43,9 +48,19 @@ describe("searchLicenseStatusFactory", () => {
     });
   });
 
-  describe("when license type is not public movers and warehouse", () => {
+  describe("when license type is health care services", () => {
+    it("returns new status client when licenseType is Health Care Services", () => {
+      const licenseType = "Health Care Services";
+      const licenseStatus = searchLicenseStatus(licenseType);
+      licenseStatus(generateLicenseSearchNameAndAddress({}), licenseType);
+      expect(stubNewSearchLicenseStatus).toHaveBeenCalled();
+      expect(stubLegacySearchLicenseStatus).not.toHaveBeenCalled();
+    });
+  });
+
+  describe("legacy search license status", () => {
```

Comment:
[sand] The previous name of this describe provided a condition. Maybe something more like this? Feel free to massage this.
```suggestion
  describe("when license type is not included in dynamics", () => {
```

Reply:
done